integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.ts

```diff
@@ -53,7 +62,28 @@ export class OpfCheckoutPaymentWrapperComponent implements OnInit, OnDestroy {
 
   protected initiatePaymentMode(): void {
     this.sub.add(
-      this.service.initiatePayment(this.selectedPaymentId).subscribe()
+      this.service.initiatePayment(this.selectedPaymentId).subscribe({
+        next: (paymentSessionData) => {
+          this.registerHostedFieldsGlobalFunctions(paymentSessionData);
+        },
+      })
     );
   }
+
+  protected registerHostedFieldsGlobalFunctions(
+    paymentSessionData: PaymentSessionData | Error
+  ) {
+    if (
```

Comment:
I would prefer to create some method like `isHostedField(args)` and use it in this statement.